tests/test_custom_templates.py

```diff
@@ -1,31 +0,0 @@
-import pytest
-import requests
-from bs4 import BeautifulSoup
-
-def test_templates_loaded(hub):
```

Comment:
I think that for this to be refactored too and run under the current testing structure, it needs the list of urls where a hub runs (the only one with multiple domains should be just mills at this moment), the url where the logo can be found and the hub template (hub-type that we're already passing).